omero/sysadmins/version-requirements.txt

```diff
@@ -322,12 +322,12 @@ If no version is provided, a suitable repository is indicated.
       - Yes
     * - 9.4
       - 6 (`postgresql <http://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/>`__), 7 (`postgresql <https://yum.postgresql.org/9.4/redhat/rhel-7-x86_64/>`__)
-      - 16.04
+      - N/A
       - Yes
       - Yes
     * - 9.5
       - 6 (`postgresql <http://yum.postgresql.org/9.5/redhat/rhel-6-x86_64/>`__), 7 (`postgresql <https://yum.postgresql.org/9.5/redhat/rhel-7-x86_64/>`__)
-      - 14.04, 16.04 (`postgresql <https://apt.postgresql.org/pub/repos/apt/>`__)
+      - 14.04 (`postgresql <https://apt.postgresql.org/pub/repos/apt/>`__), 16.04
```

Comment:
there is an inconsistency in the placement of the repo link for 9.5 compared with 9.4+9.6